config/routing.go

```diff
@@ -17,6 +17,8 @@ type Routing struct {
 
 	AcceleratedDHTClient bool
 
+	LoopbackAddressesOnLanDHT bool
```

Comment:
Can we make this a `Flag` please? 
(like we did for  `Gateway.DeserializedResponses`, `Gateway.InlineDNSLink` etc)

Making `AcceleratedDHTClient` a bool was a mistake, it now gets hardcoded in user configs forver :-/ 

If not a big pain, we should also make it a `Flag` (here or separate PR), to avoid migration headache in the future.

Reply:
See https://github.com/ipfs/kubo/pull/10384 for `AcceleratedDHTClient` as Flag.